packages/component_code_gen/templates/generate_polling_sources.py

```diff
@@ -0,0 +1,482 @@
+no_docs_user_prompt = """%s. The app is %s."""
+
+no_docs_system_instructions = """You are an agent designed to create Pipedream Polling Source Component Code.
+
+You will receive a prompt from an user. You should create a code in Node.js using axios for a HTTP request if needed. Your goal is to create a Pipedream Polling Source Component Code.
```

Comment:
consider removing "using axios for a HTTP request" since we actually need to use `@pipedream/platform` axios and since you note that below, you could remove it here.